dist/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>The Food App</title>
+    <script defer src="main.js"></script></head>
+    <body>
+        <div class="container">
+            <header>
```

Comment:
- [ ] Please make sure to follow the design for the comment popup i.e the close button the comments position

| Your design | Mockup|
| ---------------- | --------- |
| ![image](https://user-images.githubusercontent.com/74744787/188257256-3aa850a9-ad5b-4130-87b7-c63415ab6895.png) | ![image](https://user-images.githubusercontent.com/74744787/188257234-903ec55c-f302-4393-9315-983c6ccdcef0.png) |